contributing-guides/maintainers-guide.md

```diff
@@ -56,22 +56,20 @@ as a guideline for current and future maintainers.
 
 - PRs should be merged once they
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
-  (2) have the **review comments addressed**, and
-  (3) get **approved reviews by two maintainers**
-  (the second maintainer can perform the merge immediately after accepting.)
+  (2) have the **review comments addressed**,
```

Comment:
```suggestion
  (2) all **review comments are addressed**
```

Reply:
This would only make sense with the change above, so I'll resolve this to keep the conversation cleaner.